msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md

```diff
@@ -213,25 +213,30 @@ If you need to access Microsoft Graph data, configure your server-side code to:
 > 
 > * Don’t use `notifySuccess` result to return the token information to the parent page. Use `localStorage` to save the token and pass the item key via `notifySuccess`.
 
-## Known limitations
+## Obtaining consent
 
 Tenant admin consent: A simple way of [consenting on behalf of an organization as a tenant admin](/azure/active-directory/manage-apps/consent-and-permissions-overview#admin-consent) is by getting [consent from admin](/azure/active-directory/manage-apps/grant-admin-consent).
 
-You can ask for consent using the Auth API. Another approach for getting Graph scopes is to present a consent dialog using our existing [third party OAuth provider authentication approach](~/tabs/how-to/authentication/auth-tab-aad.md#navigate-to-the-authorization-page-from-your-pop-up-page). This approach involves popping up an Azure AD consent dialog box.
+Another approach for getting Graph scopes is to present a consent dialog using our existing [third party OAuth provider authentication approach](~/tabs/how-to/authentication/auth-tab-aad.md#navigate-to-the-authorization-page-from-your-pop-up-page). This approach involves popping up an Azure AD consent dialog box.
+
+You can also ask for consent using the TeamsJS [authentication](/javascript/api/@microsoft/teams-js/authentication) capability. 
 
 <details>
-<summary>To ask for additional consent using the Auth API, follow these steps:</summary>
+<summary>To obtain consent using the TeamsJS client library, follow these steps:</summary>
+
+> [!TIP]
+> The [Teams Personal Tab SSO Authentication](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/tab-personal-sso-quickstart/js/src/components/Tab.js#L64-L101) sample provides code demonstrating following steps.
 
 1. The token retrieved using `getAuthToken()` must be exchanged on the server-side using Azure AD [on-behalf-of flow](/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) to get access to those other Graph APIs. Ensure you use the v2 Graph endpoint for this exchange.
```

Comment:
Should this say "v2 **AAD** endpoint for this exchange?"

Reply:
I'm not sure! The [linked doc](https://review.learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow?branch=main&branchFallbackFrom=pr-en-us-8868#success-response-example) talks about v1- and v2-formatted tokens for Microsoft Graph. I'm not able to tell if we're talking about v2 of AAD, Graph, or OAuth. 

@iambmelt could you help interpret this phrase: *Ensure you use the v2 Graph endpoint for this exchange* Thanks!

Reply:
Confirming comment from @AE-MS above - this should be the AAD v2 endpoint